asm/melee/ft/ftcoll.s

```diff
@@ -5886,41 +5886,6 @@ lbl_8007B58C:
 /* 8007B5A4 00078184  98 07 22 1A */	stb r0, 0x221a(r7)
 /* 8007B5A8 00078188  4E 80 00 20 */	blr 
 
-.global func_8007B5AC
```

Comment:
order matters when linking, so you can't remove a function from the middle like this. it won't match

Reply:
uh oh. what do I do then?

Reply:
either leave the function there and just add the C code as an example for when the rest of the functions are matched, or convert the entire ftcoll.s to inline asm in ftcoll.c and match func_8007B5AC inplace

Reply:
ok, i added the ASM back in.

Reply:
i'll keep that in mind next time

Reply:
Currently this is just compiling the asm, and ignoring the C. You need to add both `ftcoll.c` and `ftcoll.s` to `obj_files.mk`, which might require you to port asm into the C file as-is (see other `asm` functions as examples).